.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
ISSUE: like I mentioned in the pull request for #16, please gitignore these .idea files and remove them from the pull request. If we have issues later on we can fix it on a case by case basis